app/views/shared/_header.html.erb

```diff
@@ -16,8 +16,17 @@
           </li>
 
           <li>
-            <%= link_to decorated_current_user.github_url, class: 'tooltipped tooltipped-s', 'aria-label' => 'View GitHub profile' do %>
-              <%= image_tag decorated_current_user.avatar_url(40), class: 'avatar', alt: "@#{decorated_current_user.login}", height: 20, width: 20 %>
+            <% if impersonating?(current_user, true_user) %>
+              <%= link_to stop_impersonating_stafftools_user_path(current_user), class: 'tooltipped tooltipped-s', method: :delete, 'aria-label' => "@#{decorated_true_user.login} impersonating @#{decorated_current_user.login}" do %>
```

Comment:
The tooltip text should read:

> Stop impersonating @username

The other tooltips are actions, this one should be as well.